x/concentrated-liquidity/simulation/sim_msgs.go

```diff
@@ -18,6 +18,7 @@ import (
 var PoolCreationFee = sdk.NewInt64Coin("stake", 10_000_000)
 
 func RandomMsgCreateConcentratedPool(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*clmodeltypes.MsgCreateConcentratedPool, error) {
+	rand := sim.GetSeededRand("select random seed")
```

Comment:
The seed should already be configured at the simulator test level. Please use that one.

AFAIU, the rand manager that is seeded can be grabbed from here:
https://github.com/osmosis-labs/osmosis/blob/9d9806b4f36db9516fdb7e7419e4ab74eaf24997/simulation/simtypes/simctx.go#L20

Reply:
This is where the seed value is set: https://github.com/osmosis-labs/osmosis/blob/9d9806b4f36db9516fdb7e7419e4ab74eaf24997/tests/simulator/sim_test.go#L41

This is what should control all randomness in the simulator.

If that value is constant, I should get the same results deterministically. If I change it, only then I get different results.

By initializing new seeds here, we are introducing randomness independent of that global seed value so it becomes non-trivial to control

Reply:
aah yea so it's getting random seed with "FlagSeedValue + string we pass"

Reply:
this is the function that uses FlagSeedValue and adds the string we pass(https://github.com/osmosis-labs/osmosis/blob/main/simulation/simtypes/simctx.go#L156) 

i think its fine to use string, i've changed string name to follow function name too

Reply:
So what I'm afraid is that we're resetting the global seed here.

When I made a comment originally, I just wanted to make sure that we are using a simulator global-seed that is configured at the top-level of the test that I linked.

I think we can get the `rand` associated with that seed by doing:
```
simRand := sim.GetRand()

randInt := simRand.Intn()
```